src/table/body-cell/td-element.tsx

```diff
@@ -98,10 +103,17 @@ export const TableTdElement = React.forwardRef<HTMLTableCellElement, TableTdElem
           stickyStyles.className
         )}
         onClick={onClick}
+        // Scroll cell into view when it is focused with keyboard.
+        onFocus={event => {
+          if (!event.currentTarget.contains(getMouseDownTarget())) {
+            event.target.scrollIntoView?.({ block: 'center' });
```

Comment:
Why not use our existing utility `scrollElementIntoView`?

Reply:
Thanks, will check it